lib/exercises.rb

```diff
@@ -1,19 +1,53 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n * m) where n is word and m is char 
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 However you can assume the time complexity is O(n) if each word is limited in size.